externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala

```diff
@@ -21,8 +21,9 @@ import java.time.ZoneId
 
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.{DataFrame, Dataset, Row}
+import org.apache.spark.sql.execution.HiveResult
```

Comment:
is there any compatibility issue for different Spark version ?

Reply:
The signature is stable since 3.1, and the current implementation is compatible w/ Spark 3.1 and above.